content/en/docs/components/serving/bentoml.md

```diff
@@ -0,0 +1,258 @@
++++
+title = "BentoML"
+description = "Model serving with BentoML"
+weight = 40
++++
+
+{{% stable-status %}}
+
+## Model serving with BentoML
+
+In this guide, we will demo serving a scikit-learn based iris classifier model with
+BentoML on a Kubernetes cluster. The same deployment steps is also applicable for models
+trained with other machine learning frameworks, see more BentoML examples
+[here](https://docs.bentoml.org/en/latest/examples.html).
+
+[BentoML](https://bentoml.org) is an open-source platform for high-performance ML model
+serving. It makes building production API endpoint for your ML model easy and supports
+all major machine learning training frameworks, including Tensorflow, Keras, PyTorch,
+XGBoost, scikit-learn and etc.
+
+BentoML comes with a high-performance API model server with adaptive micro-batching
+support, which achieves the advantage of batch processing in online serving. It also
+provides model management and model deployment functionality, giving ML teams an
+end-to-end model serving workflow, with DevOps best practices baked in.
+
+### Prerequisites
+
```

Comment:
Recommendation: Add an introductory sentence here, to introduce the list and to help people who may not understand the word "prerequisites":

```
Before starting this tutorial, make sure you have the following:
```